kratos/utilities/particles_utilities.h

```diff
@@ -134,6 +134,67 @@ class ParticlesUtilities
 
     }
 
+/**@brief: this function takes all the nodes in rParticlesModelPart and finds in which element they fall within
```

Comment:
Can this be moved to a cpp?

Reply:
It is templated but you are exposing a finite number of them. Do you want it to keep it like this?, TDim is 2 or 3, and TScalarType is int, unsigned it and double. Can be explicitly templated

Reply:
sorry, but i do not see any good reason for doing it ...